extensions/kill-process/src/index.tsx

```diff
@@ -82,8 +92,13 @@ export default function ProcessList() {
   const killProcess = (process: Process) => {
     exec(`kill -9 ${process.id}`);
     setFetchResult(state.filter((p) => p.id !== process.id));
-    clearSearchBar({ forceScrollToTop: true });
-    showHUD(`✅ Killed ${process.processName === "-" ? `process ${process.id}` : process.processName}`);
+    if (!multipleKills) {
+      closeMainWindow();
```

Comment:
regarding close main window, personally, I think it is not needed, but other reviewers might have different thoughts.

Reply:
I agree with this as well.

Reply:
I don't think it's needed either, for me it already closes it, so I think it just respects Raycast's settings.

Reply:
if we all agree it is not necessary to close window, I don't think multiple kills pref is needed.

the main change for this PR is to migrate from showHUD to showToast, as showHUD will close window directly so that it will be annoying to kill multiple processes.

Reply:
Oh, I didn't know that `showHUD` is actually what's closing the window. In that case, `allowMultipleKills` doesn't make sense anymore, no? With `showToast` it will always leave the window open and you can do multiple kills.

What would make sens would be a "Close window after kill" preference, default to `true`, because most of the time/people need to kill one process and that's it.

Reply:
Yep, I've changed the preference to `closeWindowAfterKill`, with a default to `true`

Reply:
@pernielsentikaer Yup, that's what we're discussing here 😄